### ISSUE TITLE
Guam observations and threshholds questions

### ISSUE THREAD

**Issue by greg-does-weather** [weather-gov/weather.gov#1726]
### Description

Quote from a forecaster in the GUM WFO:

> "Meteorologist from WFO GUM, Edwin Montvila.
> 
> (1) For the four islands in the Marianas, The observation station selected does not match the appropriate closest observation site. For example, selecting Tamuning, Guam selects Rota International Airport (PGRO) as the closest observation site, instead of Guam International Airport (PGUM). For Saipan, it selects Tinian's Airport (PGWT) instead of Saipan International Airport (PGSN).
> 
> (2) For shower and thunderstorm coverage usage, what guidelines are used for that? Here is what WFO GUM uses:
 >
> 15-24 PoP: Isolated Showers (Days 1-3), slight chance of showers (Days 4-7).
> 25-54 PoP: Scattered Showers (Days 1-3), chance of showers (Days 4-7).
>
> Currently, for 30 PoP, the 7-Day forecast and hourly forecast indicate that this is both scattered showers and thunderstorms, which implies there are scattered thunderstorms as well. The reality is that the public/zone product says scattered showers and isolated thunderstorms.
> 
> As it is hard to attend the Weather.gov 2.0 Discussions due to the time difference, would these forms be the appropriate way to provide continuous feedback, or is there another way to do so? Thank you in advance."

### Acceptance criteria

- [ ] Tests have been implemented or modified to capture the bug
- [ ] Changes have been tested for accessibility

**Comment by colinmurphy01** [comment]
Let's investigate this and then potentially move it into `blocked`

**Comment by colinmurphy01** [comment]
Let's pick this up and figure out what to do this, dependency on interop as well.

**Comment by eric-gade** [comment]
## TL;DR ##
Several of the observation stations in this area have `null` temperature readings, so it seems we are displaying the 2nd or 3rd choice for many locations.
  
## Example of the issue ##
Tamuning, Guam resolves to lat,lon `13.492,144.79` in our system, which gives us the following `points` endpoint:
https://api.weather.gov/points/13.492%2C144.79
  
That then gives us the following endpoint for the list of closest observation stations, in order:
https://api.weather.gov/gridpoints/GUM/47,50/stations
  
You will see that the top three in this case are:
1. PGUM - Guam International Airport ([observation here](https://api.weather.gov/stations/PGUM/observations?limit=1))
2. PGUA - Andersen Air Force Base ([observation here](https://api.weather.gov/stations/PGUA/observations?limit=1))
3. PGRO - Rota International Airport ([observation here](https://api.weather.gov/stations/PGRO/observations?limit=1))
  
If you look at the first two linked observations, you'll see that the **temperature reading is `null`** for both. This is disqualifying according to our rules, and therefore PGRO is the closest observation station that has a temperature reading. 
  
Given that we received this feedback a while ago, and it _still_ seems to be the case that both of these observation stations have no temperature reading, we might want to think about contacting someone / escalating. It's strange that an airport and also a military air base would not have working temperature readings.

**Comment by colinmurphy01** [comment]
@coreypieper @shadkeene do you all have a connection with someone at MADIS  to investigate this on their end?
